geomesa-convert/geomesa-convert-json/src/test/scala/org/locationtech/geomesa/convert/json/JsonConverterTest.scala

```diff
@@ -716,5 +716,91 @@ class JsonConverterTest extends Specification {
       ec.counter.getSuccess mustEqual 0
       ec.counter.getFailure mustEqual 1
     }
+
+    "parse and convert json arrays into lists" >> {
+
+      val adv = ConfigFactory.parseString(
+        """{
+          |  type-name = "adv"
+          |  attributes = [
+          |    { name = "id",    type = "Integer"       }
+          |    { name = "sList", type = "List[String]"  }
+          |    { name = "iList", type = "List[Integer]" }
+          |    { name = "dList", type = "List[Double]"  }
+          |    { name = "uList", type = "List[UUID]"    }
+          |    { name = "geom",  type = "Point"         }
+          |  ]
+          |}
+        """.stripMargin)
+
+      val advSft = SimpleFeatureTypes.createType(adv)
+
+      val jsonStr =
+        """ {
+          |    DataSource: { name: "myjson" },
+          |    Features: [
+          |      {
+          |        "id": 1,
+          |        "geometry": {"type": "Point", "coordinates": [55, 56]},
+          |        "things": [
+          |          {
+          |            "s": "s1",
+          |            "i": 1,
+          |            "d": 1.1,
+          |            "u": "12345678-1234-1234-1234-123456781234"
+          |          },
+                     {
+          |            "s": "s2",
```

Comment:
So seems like it's merging fields from a list of objects. Will it also work if you just have a plain list of e.g. ints or something?